Tools/atl/upload.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -e
+
+PX4_BINARY_FILE="$1"
+
+echo "uploading: $PX4_BINARY_FILE"
+
+PX4_BINARY_FILE_SIZE=$(stat -c%s "$PX4_BINARY_FILE")
+curl -v -F "image=@$PX4_BINARY_FILE" -H "Expect:" -H "File-Size:$PX4_BINARY_FILE_SIZE" http://192.168.42.1/cgi-bin/upload
```

Comment:
This should live somewhere in `boards/atl/mantis-edu/` as it's very specific to that board.

Reply:
Ok, I moved it.